.github/workflows/acceptance-test.yml

```diff
@@ -0,0 +1,83 @@
+name: Acceptance Test cases
+on:
+  pull_request:
+  repository_dispatch:
+    types: [ok-to-test-command]
+jobs:
+  Acceptance-Tests:
+    runs-on: self-hosted
+    if: github.event_name == 'repository_dispatch'
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          ref: 'refs/pull/${{ github.event.client_payload.pull_request.number }}/merge'
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Acceptance test cases
+        run: CGO_ENABLED=0 NUTANIX_USERNAME="admin" NUTANIX_PASSWORD="Nutanix.123" NUTANIX_INSECURE=true NUTANIX_PORT=9440 NUTANIX_ENDPOINT="10.51.144.11" NUTANIX_PE_PASSWORD="Nutanix.123" NUTANIX_PE_USERNAME="admin" NUTANIX_STORAGE_CONTAINER="80766075-f2aa-4aee-a526-63d4a047d2ab" make testacc
```

Comment:
Could we move these to Github action secrets like [here](https://github.com/nutanix/terraform-provider-nutanix/pull/325/files#diff-0c318bde7aec58c08f04a0656294734b3e1a75793b05a4443f9a635a3920157fR62)?

Reply:
Yeah makes sense